### PR TITLE
Use varint for vec length

### PR DESCRIPTION
**Pull request by simlay**

### CODE REVIEW

File: fluvio-protocol-core/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-protocol-core"
 edition = "2018"
-version = "0.1.0"
+version = "0.1.1"
```
Comment (sehz) [inline fluvio-protocol-core/Cargo.toml]: Should bump to this to 0.2 encoding/decoding algorithm has changed

**Review by sehz — CHANGES_REQUESTED** [review]
Looks good.  A minor point about version bump

File: fluvio-protocol-core/Cargo.toml (hunk 1 above)
Comment (simlay) [reply fluvio-protocol-core/Cargo.toml]: Done. I did it for all the crates in the workspace.

**Review by sehz — APPROVED** [review]

**Comment by sehz** [comment]
crates are published

**Pull request merged**